MonstercatNet.Tests/LiveApiTests.cs

```diff
@@ -175,6 +176,30 @@ public async Task Test_StreamTrackAsStream()
             Assert.IsTrue(result.Length > 0);
         }
 
+        [Test, Order(14)]
+        public async Task Test_CreatePlaylist()
+        {
+            await Api.CreatePlaylist(new Playlist()
+            {
+                Name = $"MyTestPlaylist",
+                Public = true,
+                Tracks = new PlaylistTrack[]
+                {
+                    new PlaylistTrack()
+                    {
+                        ReleaseId = Guid.Parse("09497970-9679-4ea6-930d-e1bf22cfc994"),
+                        TrackId = Guid.Parse("c8d3abc3-1668-42de-b832-b58ca6cc883f")
+                    }
+                }
+            });
+        }
+
+        [Test, Order(15)]
+        public async Task Test_DeletePlaylist()
+        {
+            await Api.DeletePlaylist("35c010c3-2529-40ed-8b87-307dc6f28d70");
```

Comment:
This test fails for me, as the playlist doesnt exist.

Reply:
I don't think there is any other way to test the API call in the `LiveApiTests` other than placing the playlist ID manually there first. 

Calling the `CreatePlaylist` API call first doesn't work either because Monstercat's delete API requires the playlist ID which is not returned from the create call.

One possible workaround would be to call the get playlists Monstercat API before testing this delete method, but that's out of the scope of this PR.

Reply:
In order for this test to work for you:
1. Create a playlist in your account (either through the API or manually through the web portal)
1. While signed in, call the [get playlists API call directly to Monstercat](https://connect.monstercat.com/v2/self/playlists)
1. Copy any of your playlist's id
1. Replace the GUID in this test with the playlist's id and run the test

Reply:
> which is not returned from the create call.
actually, you do get the information from the api. The unofficial docs are just not up to date regarding this. I have verified this with postman and you get this json as response to a successful POST

![grafik](https://user-images.githubusercontent.com/11455827/94997148-a291ad00-05a9-11eb-88c7-844f1371f7a9.png)

And as you can probably see, there is an ID provided there. I also verified that you can use it to delete the playlist.

Reply:
Great thing you verified it! Ok, will update it 👍

Reply:
@Insire what's that with this API field ? You can edit the docs or tell me and I'll do it.